akka-actor/src/main/scala/akka/io/dns/CachePolicy.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.io.dns
+
+import akka.annotation.ApiMayChange
+
+import scala.concurrent.duration.{ Duration, FiniteDuration, _ }
+
+object CachePolicy {
+
+  @ApiMayChange
+  sealed trait CachePolicy
+  @ApiMayChange
+  case object Never extends CachePolicy
+  @ApiMayChange
+  case object Forever extends CachePolicy
+  @ApiMayChange
+  final class Ttl private (val value: FiniteDuration) extends CachePolicy {
```

Comment:
why not `TTL`?

Reply:
`TTL` isn't terrible, but I prefer this style, (which is nicely defined in the Google style guide: https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)

Reply:
Both ok then.